pkg/collector/corechecks/snmp/internal/report/report_metrics.go

```diff
@@ -143,6 +145,11 @@ func (ms *MetricSender) reportColumnMetrics(metricConfig profiledefinition.Metri
 			}
 		}
 		for fullIndex, value := range metricValues {
+			_, skip := skippedIndexes[fullIndex]
+			if skip {
```

Comment:
```suggestion
			if skip && isInterfaceTableMetric(symbol.OID) {
```
To make sure we don't skip OIDs with the same index but that could be unrelated to interfaces?